packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -136,29 +138,30 @@ class IconButton extends StatelessWidget {
       currentColor = color;
     else
       currentColor = disabledColor ?? Theme.of(context).disabledColor;
-    Widget result = new Padding(
-      padding: padding,
-      child: new LimitedBox(
-        maxWidth: size,
-        maxHeight: size,
+
+    Widget result = new ConstrainedBox(
+      constraints: new BoxConstraints(minWidth: 48.0, minHeight: 48.0),
+      child: new Padding(
+        padding: padding,
         child: new ConstrainedBox(
```

Comment:
I think you can simply this to just be a `SizedBox.fromSize(size: iconSize, ...`